domains/jeffrey.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "This website is a link to my personal developer portfolio.",
+  "repo": "https://github.com/SquarePear/jeffreyharmon.me",
+  "owner": {
+    "username": "SquarePear",
+    "email": "16364318+SquarePear@users.noreply.github.com"
```

Comment:
Can you specify a different email, your twitter or some other form of getting in touch with you? This in case of any changes to is-a-dev that you need to be informed about?